docs/OncoTree-Version-Converter.md

```diff
@@ -0,0 +1,17 @@
+# OncoTree Version Converter Tool
+
+The OncoTree Version Converter tool was developed to facilitate forwards- and backwards-mapping between OncoTree release versions. 
+
+## Running the tool
+
+The OncoTree Version Converter tool can be run with the following command:
+
+```
+python ./scripts/convert_oncotree_version.py --input-file <path/to/input/file> --output-file <path/to/output/file> --source-version <source_oncotree_version> --target-version <target_oncotree_version>
+```
+
+**Options**
+- `-i | --input-file`: This is the input file path. It must contain `ONCOTREE_CODE` in the header and it must contain OncoTree codes corresponding to the `<source_oncotree_version>`.
```

Comment:
I would include a link to the file formats doc and mention this is a clinical file. I would also say that ONCOTREE_CODE is a column (or a column header)